Sources/LanguageServerProtocol/SupportTypes/ServerCapabilities.swift

```diff
@@ -458,6 +464,41 @@ public struct DocumentLinkOptions: Codable, Hashable {
   }
 }
 
+public struct SemanticTokensRangeOptions: Equatable, Hashable, Codable {
+  // Empty in the LSP 3.16 spec.
+}
+
+public struct SemanticTokensFullOptions: Equatable, Hashable, Codable {
```

Comment:
Maybe this (and `SemanticTokensRangeOptions`) should be nested inside `SemanticTokensOptions`?  They're not really useful as top-level types.